test/GitHub.InlineReviews.UnitTests/Models/DiffUtilitiesTests.cs

```diff
@@ -269,5 +315,33 @@ public void NoLineMatchesFromNoLines()
                 Assert.Equal(null, line);
             }
         }
+
+        public class TheLineReaderClass
+        {
+            [Theory]
+            [InlineData("", "", null, null)]
+            [InlineData("\n", "", null, null)]
+            [InlineData("\r\n", "", null, null)]
+            [InlineData("1", "1", null, null)]
+            [InlineData("1\n2\n", "1", "2", null)]
+            [InlineData("1\n2", "1", "2", null)]
+            [InlineData("1\r\n2\n", "1", "2", null)]
+            [InlineData("1\r\n2", "1", "2", null)]
+            [InlineData("\r", "\r", null, null)]
+            [InlineData("\r\r", "\r\r", null, null)]
+            [InlineData("\r\r\n", "\r", null, null)]
+            [InlineData("\r_\n", "\r_", null, null)]
+            public void ReadLines(string text, string line1, string line2, string line3)
```

Comment:
There should also be a test checking what happens when null is passed into the constructor.